Source/WebInspectorUI/UserInterface/Controllers/DebuggerManager.js

```diff
@@ -315,7 +315,6 @@ WI.DebuggerManager = class DebuggerManager extends WI.Object
             return WI.DebuggerManager.PauseReason.EventListener;
         case InspectorBackend.Enum.Debugger.PausedReason.Exception:
             return WI.DebuggerManager.PauseReason.Exception;
-        case InspectorBackend.Enum.Debugger.PausedReason.Fetch:
             return WI.DebuggerManager.PauseReason.Fetch;
```

Comment:
Did the return statement need to be removed as well? 😅

Reply:
oh lol oops yes will remove